userguide/sharing.rst

```diff
@@ -1092,16 +1092,17 @@ settings:
   in
   :menuselection:`Services --> SMB`.
 
-* Be careful about unchecking the
-  :guilabel:`Browsable to Network Clients` box. When this box is
-  checked (the default), other users will see the names of every share
-  that exists using Windows Explorer, but they will receive a
-  permissions denied error message if they try to access someone
-  else's share. If this box is unchecked, even the owner of the share
-  will not see it or be able to create a drive mapping for the share
-  in Windows Explorer. However, they can still access the share from
-  the command line. Unchecking this option provides limited security
-  and is not a substitute for proper permissions and password control.
+* When the :guilabel:`Browsable to Network Clients` box is checked (the 
+  default), the share will be visible in the server's browse list. If the share 
+  is is flagged as a :guilabel:`home share`, unchecking this box will hide the 
+  [homes] share so that only the dynamically generated home share with the
```

Comment:
Sorry. This was half-baked. [homes] is a special section of the smb.conf file that is generated when when the user checks the box "Use as home share". Enabling home shares in samba in FreeNAS will basically generate two shares:
\\freenas\homes
\\freenas\%U
Unchecking "browsable" for a home share makes it so that \\freenas\homes isn't visible, but \\freenas\%U will be visible.